WORKSPACE

```diff
@@ -52,13 +52,22 @@ http_archive(
     urls = ["https://github.com/capnproto/capnproto/tarball/6446b721a9860eebccf9d3c73b27610491359b5a"],
 )
 
+# Needed for boringssl
```

Comment:
This really shouldn't be necessary, or at least be handled by a `boringssl_build_deps()` function, but boringssl does not support workspace-based builds all that well.